FNPlugin/Collectors/SolarWindCollector.cs

```diff
@@ -130,9 +132,22 @@ public override void OnFixedUpdate()
                 if (!bIsEnabled)
                 {
                     strCollectingStatus = "Disabled";
-                    strStarDist = (CalculateDistanceToSun() / 1000).ToString("F2") + " km"; // trying to solve the bullshit
+                    strStarDist = UpdateDistanceInGUI(); // passes the distance to the GUI
+                    return;
+                }
+
```

Comment:
Yes, I know about magnetosphere moving most of the solar wind out of the way. I have made the simplification that atmosphere == magnetosphere, which is perhaps a wee bit optimistic but it made it easier ;) We could surely either change the check to higher altitude or make it even more complex, as you propose. But I am, sadly, not familiar with the functions in Kerbalism.